app/src/main/java/com/uc/mymovies/main/MainViewModel.kt

```diff
@@ -0,0 +1,20 @@
+package com.uc.mymovies.main
+import android.util.Log
+import androidx.lifecycle.ViewModel
+import com.uc.mymovies.service.MovieService
+
+
+class MainViewModel : ViewModel() {
+
+    var movieService: MovieService = MovieService()
+
+    init {
+        fetchMovies()
+    }
+
+    private fun fetchMovies() {
+        var result = movieService.fetchUserMovies("connorkeith")
+        Log.d("TEST-LOG: ", result.toString())
+    }
+
+}
```

Comment:
create viewmodel to let the fragments share one single viewmodel

Reply:
Ok, here's where the deleted stuff moved.